src/forms/form-render/form-render-test.component.tsx

```diff
@@ -118,10 +130,10 @@ function FormRenderTest() {
                 <Tab id="tab-form" label="Form render">
                   {isSchemaLoaded ? (
                     <div className={styles.formRenderDisplay}>
-                      <OHRIForm formJson={formInput} patientUUID={patientUUID} mode={currentMode} />
+                      <OHRIForm formJson={formInput} patientUUID={patientUUID} mode={currentFormMode} />
                     </div>
                   ) : (
-                    <p>Please enter a valid schema</p>
+                    <p>Please enter a valid schema or submit form</p>
```

Comment:
@pirupius I think valid schema and submit form are different problems.. if the schema is valid and the "form is not submitted" this message will still appear. This is confusing